packages/core/.pretierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
would you be cool with using `.prettierrc.json`? it makes it easier for different IDE's to parse the file as json